### PR TITLE
Metrics web

### PR DESCRIPTION
**Pull request by prolajumokeoni**
In this project:
- Selected the Jason placeholder API
- Built files structure  app
- Created routes and navigation 
- Created unit and integration tests
- Sorted posts  alphabetically by titles
### Video link: https://www.loom.com/share/51aba3bff9704b02a0561587327a7fd6

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @prolajumokeoni, 

**This is a great project you have going on, well-done lad**. 

   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️

- [x] Add the testing command to the readme file (set up section)

_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Posts.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+          >
+            <i className="fas fa-blog fs-1 fw-bold pl-3" />
+            {' '}
+            <PostForm post={post} />
+          </div>
+        ))}
+      </div>
+    </div>
+  );
+};
```
Comment (mke2111) [inline src/components/Posts.js]: - [x] The Home page has list of categories that could be filtered using given parameters, e.g. category name.

File: src/components/SinglePost.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+const SinglePost = ({ post }) => (
+  <div className="p-2">
+    <div className="d-flex justify-content-between px-5 fw-bold mt-1">
+      <p>Blog Post</p>
+      <NavLink to="/" className="text-decoration-none text-white">
+        <i className="fas fa-arrow-left" />
+        {' '}
+        Back
+      </NavLink>
```
Comment (mke2111) [inline src/components/SinglePost.js]: - [x] Following the given design specs is one of the best practices clients would like. The layout of the `back` element is not as given in the design specs. Give this a fix. 😄
Screenshot from 2021-10-14 16-36-30
Screenshot from 2021-10-14 16-37-18

File: src/components/Posts.js (hunk 1 above)
Comment (prolajumokeoni) [reply src/components/Posts.js]: Thank you for your review @mke2111, I have implemented the changes

**Review by mke2111 — APPROVED** [review]
### Hi @prolajumokeoni, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉** 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**